htdocs/commande/list.php

```diff
@@ -1741,8 +1741,9 @@
 			Commande::STATUS_VALIDATED=>$langs->trans("StatusOrderValidated"),
 			Commande::STATUS_SHIPMENTONPROCESS=>$langs->trans("StatusOrderSentShort"),
 			Commande::STATUS_CLOSED=>$langs->trans("StatusOrderDelivered"),
+			-4=>$langs->trans("StatusOrderValidatedShort").'+'.$langs->trans("StatusOrderSentShort"),
 			-3=>$langs->trans("StatusOrderValidatedShort").'+'.$langs->trans("StatusOrderSentShort").'+'.$langs->trans("StatusOrderDelivered"),
-			-2=>$langs->trans("StatusOrderValidatedShort").'+'.$langs->trans("StatusOrderSentShort"),
+			-2=>$langs->trans("StatusOrderToProcessShort"),
```

Comment:
Filter executed when using -2 is c.fk_statut IN (1,2) 
So we make a filter on 2 different status. So we should see the 2 status on the combo list.

Reply:
My bad, did not see the sql condition changed from V16... closing the request. :)